### PR TITLE
[Keyboard] Re-add rart67

### PR DESCRIPTION
**Pull request by alabahuy**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by alabahuy** [comment]
@noroadsleft this pcb have 2x 1u, you can see again layout in [RART67](https://github.com/alabahuy/RART/tree/master/RART67)

File: keyboards/rart/rart67/config.h [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID		  0x414C  // "AL"
+#define PRODUCT_ID 		0x0067  // "67"
+#define DEVICE_VER 		0x0001
```
Comment (noroadsleft) [inline keyboards/rart/rart67/config.h:25]: Just a tabs-to-spaces change.

```suggestion
#define VENDOR_ID       0x414C  // "AL"
#define PRODUCT_ID      0x0067  // "67"
#define DEVICE_VER      0x0001
```

**Comment by alabahuy** [comment]
@noroadsleft for the error PR Lint keyboard its not my keyboard, what should i do?

File: keyboards/rart/rart67/config.h (hunk 1 above)
Comment (alabahuy) [reply keyboards/rart/rart67/config.h]: done

**Comment by noroadsleft** [comment]
> for the error PR Lint keyboard its not my keyboard, what should i do?

You can ignore it. It's a CI test that we haven't completed yet.

Everything's good to go here, thanks!

**Pull request merged**